plugins/importexport/datacite/scheduledTasks.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  * plugins/importexport/datacite/scheduledTasks.xml
+  *
+  * Copyright (c) 2013-2015 Simon Fraser University Library
+  * Copyright (c) 2003-2015 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * DataCite Plugin Automatic Deposit
+  *
+  * This file describes the schedule task that will send deposits to DataCite in the background.
+  *
+  -->
+
+<!DOCTYPE scheduled_tasks SYSTEM "../../../lib/pkp/dtd/scheduledTasks.dtd">
+
+<scheduled_tasks>
+	<task class="plugins.importexport.datacite.DataciteInfoSender">
+		<descr>Send DOI deposit to DataCite server.</descr>
+		<frequency minute="0"/>
```

Comment:
If this is used by the acron plugin with frequency 0, then every website request will trigger the scheduled task. I think that 24 hours might be a better value.